app/models/auth_source.rb

```diff
@@ -17,6 +17,11 @@
 
 class AuthSource < ActiveRecord::Base
   include Authorizable
+
+  attr_accessible :name, :host, :tls, :port, :server_type, :account,
+      :base_dn, :groups_base, :ldap_filter, :onthefly_register,
```

Comment:
I think many of these ought to only be in the LDAP subclass as they're unused in the base.